packages/sources/bea/README.md

```diff
@@ -0,0 +1,52 @@
+# Chainlink External Adapter for Bea
+
+### Environment Variables
+
+| Required? |  Name   |                            Description                             | Options | Defaults to |
+| :-------: | :-----: | :----------------------------------------------------------------: | :-----: | :---------: |
+|           | API_KEY | An API key that can be obtained from the data provider's dashboard |         |             |
+
+---
+
+### Input Parameters
+
+| Required? |   Name   |     Description     |        Options         | Defaults to |
+| :-------: | :------: | :-----------------: | :--------------------: | :---------: |
+|           | endpoint | The endpoint to use | [price](#Bea-Endpoint) |    price    |
+
+---
+
+## Bea Endpoint
```

Comment:
The endpoint name here doesn't match what the endpoint name is in the code.

Anyways, can we call this the AVG endpoint? And also add the description that we will get the average of the last N values.